metaflow/datatools/s3op.py

```diff
@@ -128,11 +263,11 @@ def process_urls(mode, urls, verbose, num_workers):
               file=sys.stderr)
 
     start = time.time()
-    start_workers(mode, urls, num_workers)
+    sz_results = start_workers(mode, urls, num_workers)
     end = time.time()
 
     if verbose:
-        total_size = sum(size for url, size in urls)
+        total_size = sum(sz for sz in sz_results if sz is not None and sz > 0)
         bw = total_size / (end - start)
         print('%sed %d files, %s in total, in %d seconds (%s/s).'\
               % (mode.capitalize(),
```

Comment:
Nit that we should address later - we might not always be successful in downloading all the files (--no-allow-missing and what not)

Reply:
those errors should be caught later ideally. At least the behavior here should not have changed.